guides/release-notes.md

```diff
@@ -7,6 +7,23 @@ We love hearing your feedback! You can help us improve the docs by reporting any
 ![Was this helpful, yes/no buttons](../media/feedback.png)
 
 ---
+### New: Dashboard Builder Elements
+
+`Wix Editor Elements ($w)`
+
+Dashboard elements provide the following functionality to your user's Dashboard:
```

Comment:
> Dashboard elements provide the following functionality to your user's Dashboard:

You can now add these elements to dashboard pages in your Wix site or in Wix Blocks: